scripts/update-examples-dep.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
@mehulkar I could have sworn you already had a script similar to this - is it here and I'm missing it?

Reply:
https://github.com/vercel/turbo/pull/5922